Classes/Views/Styles.swift

```diff
@@ -73,8 +73,8 @@ enum Styles {
         static let button = UIFont.systemFont(ofSize: Styles.Sizes.Text.button)
         static let headline = UIFont.boldSystemFont(ofSize: Styles.Sizes.Text.headline)
         static let smallTitle = UIFont.boldSystemFont(ofSize: Styles.Sizes.Text.smallTitle)
-        static let code = UIFont(name: "Courier", size: Styles.Sizes.Text.body)!
-        static let secondaryCode = UIFont(name: "Courier", size: Styles.Sizes.Text.secondary)!
+        static let code = UIFont(name: "Menlo", size: Styles.Sizes.Text.body)!
```

Comment:
Any particular reason for the change of font? Also is Menlo monospaced?

Reply:
`Menlo` is monospaced, yep! Used by Apple prior to `SF Mono` in Xcode I think.

Reply:
Which reminds me... why, in that case, would we go for `Menlo` over `SF Mono`?

Reply:
I just thought it looked nice then Courier :) Shall I change it to SF Mono?